### PR TITLE
Safe

### PR DESCRIPTION
**Pull request by larskluge**
New page `/safe` with an ability to embed inkpads easily. 

Also a little refactoring of proposal inkpad embeds.

### CODE REVIEW

File: lib/helpers/proposals.coffee [hunk 1]
```diff
@@ -28,11 +28,5 @@ exports.store = (rq, rs, next) ->
...
   proposal = rs.locals.proposal
   id = proposal.get('inkpad')
-
-  inkpad(id)
-    .then (result) ->
-      md = markdown(result[id] or '')
-      (rs.locals.bodyClasses ||= []).push('has-team') if md.metadata.team
-      rs.locals.inkpad = md
-      next()
-    .catch next
```
Comment (mondoreale) [inline lib/helpers/proposals.coffee]: Once we remove these line we can also remove
```coffeescript
inkpad = require('./inkpads')
markdown = require('inkpad-markdown')
```

**Comment by mondoreale** [comment]
Looking good; think it'll fly. 🚀

File: lib/helpers/proposals.coffee (hunk 1 above)
Comment (larskluge) [inline lib/helpers/proposals.coffee]: Good catch!

**Comment by larskluge** [comment]
Cheers!

**Pull request merged**